.travis.yml

```diff
@@ -2,6 +2,7 @@ language: python
 
 python:
   - "2.7"
+  - "3.8"
```

Comment:
(Note that if you go with my suggestion in requirements, you'll have to change this to PY3.7, see https://github.com/tensorflow/tensorflow/issues/39768#issuecomment-632799481)

Reply:
ERROR: Could not find a version that satisfies the requirement tensorflow==1.15.4 (from -r requirements.txt (line 7)) (from versions: 2.2.0rc1, 2.2.0rc2, 2.2.0rc3, 2.2.0rc4, 2.2.0, 2.2.1, 2.3.0rc0, 2.3.0rc1, 2.3.0rc2, 2.3.0, 2.3.1)
ERROR: No matching distribution found for tensorflow==1.15.4 (from -r requirements.txt (line 7))

Note that on my localhost this doesn't work:

+ python model_bias_analysis_test.py
2020-10-19 14:59:35.880364: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcudart.so.10.1'; dlerror: libcudart.so.10.1: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /usr/lib/mesa-diverted/x86_64-linux-gnu:/usr/lib/x86_64-linux-gnu/mesa:/usr/lib/x86_64-linux-gnu/dri:/usr/lib/x86_64-linux-gnu/gallium-pipe
2020-10-19 14:59:35.880436: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
Traceback (most recent call last):
  File "model_bias_analysis_test.py", line 8, in <module>
    import model_bias_analysis as mba
  File "/usr/local/google/home/sorenj/git/unintended-ml-bias-analysis/unintended_ml_bias/model_bias_analysis.py", line 28, in <module>
    import matplotlib.pyplot as plt
  File "/usr/local/google/home/sorenj/git/unintended-ml-bias-analysis/.py3/lib/python3.8/site-packages/matplotlib/pyplot.py", line 115, in <module>
    _backend_mod, new_figure_manager, draw_if_interactive, _show = pylab_setup()
  File "/usr/local/google/home/sorenj/git/unintended-ml-bias-analysis/.py3/lib/python3.8/site-packages/matplotlib/backends/__init__.py", line 62, in pylab_setup
    backend_mod = __import__(backend_name, globals(), locals(),
  File "/usr/local/google/home/sorenj/git/unintended-ml-bias-analysis/.py3/lib/python3.8/site-packages/matplotlib/backends/backend_tkagg.py", line 4, in <module>
    from . import tkagg  # Paint image to Tk photo blitter extension.
  File "/usr/local/google/home/sorenj/git/unintended-ml-bias-analysis/.py3/lib/python3.8/site-packages/matplotlib/backends/tkagg.py", line 5, in <module>
    from six.moves import tkinter as Tk
  File "/usr/local/google/home/sorenj/git/unintended-ml-bias-analysis/.py3/lib/python3.8/site-packages/six.py", line 92, in __get__
    result = self._resolve()
  File "/usr/local/google/home/sorenj/git/unintended-ml-bias-analysis/.py3/lib/python3.8/site-packages/six.py", line 115, in _resolve
    return _import_module(self.mod)
  File "/usr/local/google/home/sorenj/git/unintended-ml-bias-analysis/.py3/lib/python3.8/site-packages/six.py", line 82, in _import_module
    __import__(name)
ModuleNotFoundError: No module named 'tkinter'